eva_submission/submission_qc_checks.py

```diff
@@ -138,24 +138,47 @@ def get_files_from_ftp(self, project_accession):
         ftp.cwd(f'pub/databases/eva/{project_accession}')
         return ftp.nlst()
 
-    def check_if_job_completed_successfully(self, file_path):
+    def get_job_launched_and_completed_text(self, job_type):
+        if job_type == 'accession':
+            return ['Job: [SimpleJob: [name=CREATE_SUBSNP_ACCESSION_JOB]] launched'], \
+                ['Job: [SimpleJob: [name=CREATE_SUBSNP_ACCESSION_JOB]] completed']
+        elif job_type == 'variant_load':
+            return ['Job: [FlowJob: [name=genotyped-vcf-job]] launched',
+                    'Job: [FlowJob: [name=aggregated-vcf-job]] launched'], \
+                ['Job: [FlowJob: [name=genotyped-vcf-job]] completed',
+                 'Job: [FlowJob: [name=aggregated-vcf-job]] completed']
+        elif job_type == 'clustering':
+            return ['Job: [SimpleJob: [name=STUDY_CLUSTERING_JOB]] launched'], \
+                ['Job: [SimpleJob: [name=STUDY_CLUSTERING_JOB]] completed']
+        elif job_type == 'clustering_qc':
+            return ['Job: [SimpleJob: [name=NEW_CLUSTERED_VARIANTS_QC_JOB]] launched'], \
+                ['Job: [SimpleJob: [name=NEW_CLUSTERED_VARIANTS_QC_JOB]] completed']
+        elif job_type == 'vcf_extractor':
+            return ['Job: [SimpleJob: [name=EXPORT_SUBMITTED_VARIANTS_JOB]] launched'], \
+                ['Job: [SimpleJob: [name=EXPORT_SUBMITTED_VARIANTS_JOB]] completed']
+        elif job_type == 'remapping_ingestion':
+            return ['Job: [SimpleJob: [name=INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB]] launched'], \
+                ['Job: [SimpleJob: [name=INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB]] completed']
+        elif job_type == 'backpropagation':
+            return ['Job: [SimpleJob: [name=BACK_PROPAGATE_NEW_RS_JOB]] launched'], \
+                ['Job: [SimpleJob: [name=BACK_PROPAGATE_NEW_RS_JOB]] completed']
```

Comment:
I think it would be easier to read as a map
```suggestion
job_launched_and_completed_text_map = { 
    'accession': (
        {'Job: [SimpleJob: [name=CREATE_SUBSNP_ACCESSION_JOB]] launched'}, 
        {'Job: [SimpleJob: [name=CREATE_SUBSNP_ACCESSION_JOB]] completed'}
    ),
    'variant_load': (
        {'Job: [FlowJob: [name=genotyped-vcf-job]] launched', 'Job: [FlowJob: [name=aggregated-vcf-job]] launched'},
        {'Job: [FlowJob: [name=genotyped-vcf-job]] completed', 'Job: [FlowJob: [name=aggregated-vcf-job]] completed'}
    ),
    'clustering':(
            {'Job: [SimpleJob: [name=STUDY_CLUSTERING_JOB]] launched'},
            {'Job: [SimpleJob: [name=STUDY_CLUSTERING_JOB]] completed'}
    ),
    'clustering_qc':(
        {'Job: [SimpleJob: [name=NEW_CLUSTERED_VARIANTS_QC_JOB]] launched'},
        {'Job: [SimpleJob: [name=NEW_CLUSTERED_VARIANTS_QC_JOB]] completed'}
    ),
    'vcf_extractor':(
        {'Job: [SimpleJob: [name=EXPORT_SUBMITTED_VARIANTS_JOB]] launched'},
        {'Job: [SimpleJob: [name=EXPORT_SUBMITTED_VARIANTS_JOB]] completed'}
    ),
    'remapping_ingestion':(
        {'Job: [SimpleJob: [name=INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB]] launched'},
        {'Job: [SimpleJob: [name=INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB]] completed'}
    ),
    'backpropagation':(
        {'Job: [SimpleJob: [name=BACK_PROPAGATE_NEW_RS_JOB]] launched'},
        {'Job: [SimpleJob: [name=BACK_PROPAGATE_NEW_RS_JOB]] completed'}
    )
}
```

Reply:
updated